CanlabCore/@fmridisplay/addpoints.m

```diff
@@ -178,7 +178,7 @@
             case 'axial'
                 whcol = 3;
                 
-            case {'sagg', 'sagittal', 'saggital'}
+            case {'sagg', 'sagittal', 'sagittal'}
```

Comment:
@jc2 , following your comment in the main thread (sorry -- missed), you want to expand this to contain all of them and then we need to ignore the line to not fix them up, so smth like

```suggestion
            case {'sagg', 'sagittal', 'sagittal', 'saggital'}  # codespell:ignore
```
?